### ISSUE TITLE
Detect getters and setters stricter

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#1853]
This code currently does not produce any violations:

```python
class SomeExample(object):
    my_value = 1

    def get_value(self) -> int:
        return self.my_value

    def set_value(self, value: int):
        self.my_value = value
```

But, it should. We should check if there are two methods with `get_` and `set_` with the same name and raise a violation.

Related #1054 
Related #1185

**Comment by shwetana** [comment]
Hello, Greetings from me! I am interested to contribute.  Can I continue with the code??

**Comment by sobolevn** [comment]
@shwetana sorry, but this task is already implemented! See #1905 (Sorry, I have missed to link the PR properly to this task).

Please, feel free to choose any other task in our bugtracker! And feel free to ask me any questions you have, I am here to help! 🙂
